packages/markers/src/browser/problem/problem-tree-model.ts

```diff
@@ -65,6 +65,11 @@ export class ProblemTree extends MarkerTree<Diagnostic> {
         if (columnNumber !== 0) {
             return columnNumber;
         }
+        // Sort by owner in alphabetical order
```

Comment:
minor (for consistency):

```suggestion
        // Sort by owner in alphabetical order.
```